circle.yml

```diff
@@ -3,15 +3,19 @@ machine:
 
 dependencies:
   post:
+    - sudo apt-get update
     - >
       sudo apt-get install -y
+      upstart
```

Comment:
Please see https://github.com/Antergos/lightdm-webkit2-greeter/pull/74/commits/f1079f8a0aee0f9a9d26df218ec71daa143c4d2b for a detailed explanation. 

Also note that I've reverted this change now in https://github.com/Antergos/lightdm-webkit2-greeter/pull/74/commits/352460fd99a93463eb886bfcf2d984e5a2c0c5f4. At first I thought I could fix the mentioned error by upgrading upstart...